ksql-parser/src/main/java/io/confluent/ksql/parser/tree/QuerySpecification.java

```diff
@@ -36,7 +38,7 @@
 
   public QuerySpecification(
       Select select,
-      Relation into,
+      Pair<Relation, Boolean> into,
```

Comment:
nit: new line. also elsewhere

Reply:
`Pair<Relation, Boolean> into,` is one argument. Can you elaborate more?